plugins/crd/plugin_impl_crd.go

```diff
@@ -126,7 +125,7 @@ func networkServiceValidation() *apiextv1beta1.CustomResourceValidation {
 							Type:        "string",
 							MaxLength:   &maxLength,
 							Description: "NetworkService Name",
-							Pattern:     `^[a-zA-Z0-9]+\-[a-zA-Z0-9]*$`,
+							Pattern:     `^[a-zA-Z0-9]+[\-a-zA-Z0-9]*$`,
```

Comment:
Is this change related, or did the unit test catch this? Same for lines 156, 182 and 196.

Reply:
Running new unit test I found a bug in original regex expression, so it is a new corrected pattern.